docs/Tutorial.md

```diff
@@ -877,13 +877,13 @@ For this tutorial, since there is no public authentication API, we can use a fak
 
 The `authProvider` must expose 5 methods, each returning a `Promise`:
 
-```js
+```ts
 // in src/authProvider.ts
+import { AuthProvider } from "react-admin";
 
-// TypeScript users must reference the type: `AuthProvider`
-export const authProvider = {
+export const authProvider: AuthProvider = {
   // called when the user attempts to log in
-  login: ({ username }) => {
+  login: ({ username }: { username: string }) => {
```

Comment:
As you already declared `authProvider` as an `AuthProvider`, you don't need to provide more types. Same for all methods